src/SFML/Window/Unix/WindowImplX11.cpp

```diff
@@ -843,6 +844,28 @@ void WindowImplX11::setSize(const Vector2u& size)
 }
 
 
+////////////////////////////////////////////////////////////
+void WindowImplX11::setMinimumSize(const std::optional<Vector2u>& minimumSize)
+{
+    // Do nothing if resizing is disabled
+    if (m_useSizeHints)
+        return;
+
+    WindowImpl::setMinimumSize(minimumSize);
```

Comment:
I am wondering if this call should happen even when resizing is disabled, so that asserts about a potential following setMaximumSize call would be be triggered like on other platforms.

Reply:
If we don't disable `WindowImpl::setMinimumSize(minimumSize);` then a non-resizeable window suddenly becomes resizable by the user.

Reply:
I think this is what `setWindowSizeConstraints()` would do, but not `WindowImpl::setMinimumSize(minimumSize)`?
Anyways, maybe we can call it good enough for now and polish when we implement more window states and styles.